### PR TITLE
quickstart.md updated

### PR DESCRIPTION
**Pull request by Aparnarr**
Signed-off-by: Aparna <Aparna.Rr@emc.com>

1. Format fix
 2. Minor editions

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -30,18 +32,18 @@ To add the Pravega connector dependencies to you project, add the following entr
...
+From Flink's perspective, the connector to Pravega is part of the streaming application (not part of Flink's core runtime), so the connector code must be part of the application's code artifact (JAR file). Typically, a Flink application is bundled as a _`fat-jar`_ or _`uber-jar_`, such that all its dependencies are embedded.
 
-If you have set up the project via the above linked templates/guides, everything should be set up properly.
+ - The project set up should have been a success, if you have used the above linked [templates/guides](#creating-a-flink-stream-processing-project).
 
-If you set up a application's project and dependencies manually, you need to make sure that it builds a "jar with dependencies*, to include both the application and the connector classes.
+ - If you set up a application's project and dependencies manually, you need to make sure that it builds a _jar with dependencies_, to include both the application and the connector classes.
```
Comment (Aparnarr) [inline quickstart.md:51]: please check the last line, _jar with dependencies_,

File: quickstart.md [hunk 2]
```diff
@@ -8,19 +8,21 @@ You may obtain a copy of the License at
...
+**Note**: _You can skip this step if you have a streaming project set up already._
+
+Please use the following project templates and setup guidelines, to set up a stream processing project with Apache Flink:
 
-The easiest way to set up a stream processing project with Apache Flink is to use these project templates and setup guidelines:
   - [Project template for Java](https://ci.apache.org/projects/flink/flink-docs-stable/quickstart/java_api_quickstart.html)
   - [Project template for Scala](https://ci.apache.org/projects/flink/flink-docs-release-1.6/quickstart/scala_api_quickstart.html)
 
-After that, please follow the section below to add the Flink Pravega connectors to the project.
+Once after the set up, please follow the below section to add the **Flink Pravega connectors** to the project.
```
Comment (vijikarthi) [inline quickstart.md]: > Once after the set up, please follow the below section to add the **Flink Pravega connectors** to the project.
Please change it to 
> Once after the set up, please follow the below instructions to add the **Flink Pravega connectors** to the project.

File: quickstart.md [hunk 3]
```diff
@@ -8,19 +8,21 @@ You may obtain a copy of the License at
...
 
-## Add the connector dependencies
+## Add the Connector Dependencies
 
-To add the Pravega connector dependencies to you project, add the following entry to your project file (for example `pom.xml` for Maven):
+To add the Pravega connector dependencies to our project, use the following entry to your project file: (For example, `pom.xml` for Maven)
```
Comment (vijikarthi) [inline quickstart.md]: Please change it to like below
> To add the Pravega connector dependencies to your project, add the following entry to your project file: (For example, `pom.xml` for Maven)

File: quickstart.md [hunk 4]
```diff
@@ -30,18 +32,18 @@ To add the Pravega connector dependencies to you project, add the following entr
...
 
 ```
 ./gradlew clean install
 ```
 
-## Running / deploying the application
+## Running / Deploying the Application
 
-From Flink's perspective, the connector to Pravega is part of the streaming application (not part of Flink's core runtime), so the connector code must be part of the application's code artifact (JAR file).   Typically, a Flink application is bundled as a _fat-jar_ (also known as an _uber-jar_) such that all its dependencies are embedded.
+From Flink's perspective, the connector to Pravega is part of the streaming application (not part of Flink's core runtime), so the connector code must be part of the application's code artifact (JAR file). Typically, a Flink application is bundled as a _`fat-jar`_ or _`uber-jar_`, such that all its dependencies are embedded.
```
Comment (vijikarthi) [inline quickstart.md]: Let's use the original format  

_fat-jar_ (also known as an _uber-jar_)

**Review by vijikarthi — CHANGES_REQUESTED** [review]

File: quickstart.md (hunk 2 above)
Comment (Aparnarr) [reply quickstart.md]: Accepted suggestion

File: quickstart.md (hunk 3 above)
Comment (Aparnarr) [reply quickstart.md]: Accepted suggestion

File: quickstart.md (hunk 4 above)
Comment (Aparnarr) [reply quickstart.md]: Accepted suggestion

**Review by vijikarthi — APPROVED** [review]

**Pull request merged**